packs/age-of-ashes-bestiary/book-2-cult-of-cinders/nketiah.json

```diff
@@ -1746,7 +1746,7 @@
                     "die": "d4"
                 },
                 "description": {
-                    "value": "<p>This oak branch grows leaves in spring that change color in autumn and shed in winter. While wielding it, you gain a +2 circumstance bonus to checks to identify plants and fungi.</p>\n<hr />\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> You expend a number of charges from the staff to cast a spell from its list.</p>\n<ul>\n<li><strong>Cantrips</strong> @UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</li>\n<li><strong>1st</strong> @UUID[Compendium.pf2e.spells-srd.Item.Runic Body], @UUID[Compendium.pf2e.spells-srd.Item.Runic Weapon]</li>\n<li><strong>2nd</strong> @UUID[Compendium.pf2e.spells-srd.Item.Entangling Flora], @UUID[Compendium.pf2e.spells-srd.Item.Oaken Resilience], @UUID[Compendium.pf2e.spells-srd.Item.One with Plants], @UUID[Compendium.pf2e.spells-srd.Item.Shape Wood]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed ranks of all listed spells.</p>"
+                    "value": "<p>This oak branch grows leaves in spring that change color in autumn and shed in winter. While wielding it, you gain a +2 circumstance bonus to checks to identify plants and fungi.</p>\n<ul>\n<li><strong>Cantrips</strong> @UUID[Compendium.pf2e.spells-srd.Item.Tangle Vine]</li>\n</ul>\n<hr />\n<p><strong>Craft Requirements</strong> Supply one casting of all listed ranks of all listed spells.</p>"
```

Comment:
Looks like the Tangle Vine cantrip didn't get converted.

Reply:
It looks like I also need to support the plural form of Cantrips.